versioned_docs/version-1.2/quickstarts/ratify-on-azure.md

```diff
@@ -491,26 +491,26 @@ Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepoli
     apiVersion: config.ratify.deislabs.io/v1beta1
     kind: Verifier
     metadata:
-    name: verifier-notation
+      name: verifier-notation
     spec:
-    name: notation
-    artifactTypes: application/vnd.cncf.notary.signature
-    parameters:
+      name: notation
+      artifactTypes: application/vnd.cncf.notary.signature
+      parameters:
         verificationCertStores:
         certs:
-            - keymanagementprovider-akv
+          - keymanagementprovider-akv
         trustPolicyDoc:
         version: "1.0"
         trustPolicies:
-            - name: default
```

Comment:
The rest of this yaml file needs to be indented properly once the above two indentations are corrected.